webpack/assets/javascripts/react_app/components/ForemanModal/helpers.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ForemanModal from '.';
+
+export function extractModalNodes(children) {
```

Comment:
AFAIK the current convention is always using arrow syntax for functions unless it is not possible.